### ISSUE TITLE
Please provide more docs and code samples

### ISSUE THREAD

**Issue by saimeunt** [kisi-inc/client-js#12]
Hi,

Not really an issue but this package is kinda hard to use as is because of little to no docs about how to make something useful like unlocking something.

In particular, I'd like to know for how long is the authentication token valid, do I need to re-authenticate before every call to the API or do I need to refresh it every hour just like OAuth refresh tokens mechanism?

Next can you provide more code samples, to make something useful I had to reverse-engineer your slack command integration (/kisi) and understand what needed to be sent to the "webhooks/call" API endpoint.

```
const slackUserId = '...';
const lock = '...';
kisiClient.post('webhooks/call', {
  token: '...',
  user_id: slackUserId,
  command: '/kisi',
  text: `unlock ${lock}`
}).then(() => {
  console.log('success!');
}).catch(() => {
  console.log('failure!');
});
```

Thanks!

**Comment by lemavri** [comment]
I was able to make it work by taking a look at their Java API: https://github.com/kisi-inc/client-java/blob/master/KisiApi/src/de/kisi/api/KisiApiInterface.java

**Comment by ce07c3** [comment]
Sorry about the missing docs, we'll be updating it soon!

**Comment by ce07c3** [comment]
Closed in favor of #18.
